lib/ui/lobby.js

```diff
@@ -0,0 +1,87 @@
+import { el, list } from 'redom';
+
+import { dispatch } from './dispatch';
+
+
+class GameInfo {
+  constructor() {
+    this.el = el('tr',
+      this.gameModeDisplay = el('td'),
+      this.playerCountDisplay = el('td')
+    );
+
+    this.el.onclick = ev => {
```

Comment:
"Expected parentheses around arrow function argument having a body with curly braces"
Would you mind running eslint on this?